libs/design/src/molecules/sidebar/sidebar/sidebar-theme.scss

```diff
@@ -4,10 +4,11 @@
 	$tertiary: map-get($theme, tertiary);
 	$base: daff-map-deep-get($theme, 'core.base');
 	$font-color: daff-map-deep-get($theme, 'core.font-color');
+	$shadow-color: rgba(128, 128, 128, 0.1);
 
 	.daff-sidebar {
 		background: $base;
-		box-shadow: 6px 1px 9px 0px rgba(128, 128, 128, 0.1);
+		box-shadow: 6px 1px 9px 0px $shadow-color;
```

Comment:
`rgba(daff-color($daff-gray, 60), 0.1);`

Reply:
This is outside the scope of this PR, resolving for sanity.